README.md

```diff
@@ -53,4 +53,25 @@ Then, update your **package.json** with the following:
 }
 ```
 
+## Extending/Replacing the script configuration
+
+To extend or replace the scripts configurations, you have to create an **.exos.config.js** file exporting the following:
+
+```js
+module.exports = {
+  scripts: {
+    lint: (config, envs) => {
```

Comment:
Why to pass env variables if you can access them anywhere with `process.env`?

Reply:
It's actually an object. In the future, it can have more stuff in it.

Reply:
Ok. But what is the benefit of it?